Created a cardano chart

andyfiorini opened the pull request.

andyfiorini committed:
Created cardano chart

valo committed:
Delete .gitignore

valo commented on cardano/templates/deployment.yaml:
```diff
@@ -0,0 +1,58 @@
+      - name: data
+      {{- if .Values.persistence.enabled }}
+        persistentVolumeClaim:
+          claimName: cardano-data #{{ .Values.persistence.existingClaim | default (include "stellar-node.fullname" .) }}
```
Please name the name of the PVC configurable. Also I don't see the definition of the PersistenceVolumeClaim. Shouldn't it be included?

andyfiorini committed:
Removed fixed definition for PVC

Removed fixed definition for PVC and # sign, in order to execute the substitution.

valo commented on cardano/templates/deployment.yaml:
```diff
@@ -0,0 +1,58 @@
+      - name: data
+      {{- if .Values.persistence.enabled }}
+        persistentVolumeClaim:
+          claimName: {{ .Values.persistence.existingClaim | default (include "stellar-node.fullname" .) }}
```
In the common case we don’t have an existing PVC to put here. I think there should be a PVC definition as part of this chart. See 
https://github.com/santiment/helm-charts/blob/master/stellar-node/templates/pvc.yaml

andyfiorini committed:
Update deployment.yaml

Modified name of default for claim name.

andyfiorini commented:
Ok, I will add the pvc.yaml in another pr.  Is that ok?  

I don't know how to add a new file to this pr now.

Regards.

valo commented:
It should be possible to just push to your forked repo and the PR will update.

To speed up things I'll merge this one now
